src/Components/Input/types.ts

```diff
@@ -0,0 +1,5 @@
+export interface IInput {
+    todo:string,
+    setTodo:React.Dispatch<React.SetStateAction<string>>,
```

Comment:
Just a tip no request for changes

You can import the type Dispatch directly from React, so you don't always need to put React.. The same applies to SetStateAction.

Reply:
Thanks for the tip Mateus, I'll look into that!